pkg/controller/storagecluster/reconcile.go

```diff
@@ -190,6 +190,13 @@ func (r *ReconcileStorageCluster) reconcilePhases(
 		}
 	}
 
+	if err := validateArbiterSpec(instance, reqLogger); err != nil {
+		instance.Status.Phase = statusutil.PhaseError
+		r.client.Update(context.TODO(), instance)
```

Comment:
Error needs to be handled.

Reply:
I am doing this intentionally, I just want to update it, if it does not update it I don't care much about it.
Moreover, If we are going to handle update err then we will miss arbiter err and I do not want to club them together.

Reply:
Well this approach isn't going to make the linter happy.

I'd suggest this if your intent is to ignore the error
```suggestion
		_ = r.client.Update(context.TODO(), instance)
```

Reply:
Agreed. Be explicit (by using `_ :=`when ignoring return values, in particular errors.

Reply:
Done